main.css

```diff
@@ -0,0 +1,97 @@
+.heading {
+    font-family: 'Apple-Chancery', cursive;
+    font-style: italic;
+    font-size: 60px;
```

Comment:
try using "em" and "rem" instead of "px" for units. Read about them if you don't know what they are - they help with doing responsive design and make it scale easier